src/SFA.DAS.Commitments.Database/StoredProcedures/GetApprenticeshipsByULN.sql

```diff
@@ -0,0 +1,16 @@
+﻿CREATE PROCEDURE [dbo].[GetApprenticeshipsByULN]
+(
+  @ULN NVARCHAR(50)
+)
+AS
+
+SELECT 
+	s.*
+	FROM ApprenticeshipSummary s
+	WHERE s.ULN = @ULN
+	ORDER BY s.FirstName asc, s.LastName asc;
```

Comment:
To minimise work on the azure sql server, we should perform the ordering jit, probably in the website.

Reply:
Sure make sense